iocore/net/I_NetVConnection.h

```diff
@@ -221,12 +235,17 @@ struct NetVCOptions {
   {
     if (&that != this) {
       sni_servername    = nullptr; // release any current name.
+      ssl_servername    = nullptr;
```

Comment:
Something is wrong here. If `ssl_servername` is set to `nullptr`, why `release` it later? There's nothing there. This applies to `sni_servername` as well. Further, even if not set to `nullptr`, what is accomplished by the `release`?

Outside scope, but we should change this to support move semantics to avoid making extra copies of the source strings.

Reply:
Yes, it seemed odd to me as well.  I was following the existing pattern.